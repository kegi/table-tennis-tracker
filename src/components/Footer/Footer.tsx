import React, { Component } from 'react'

import './scss/footer.scss'

export default class Footer extends Component {
  render () {
    return (
      <footer id="footer">

        &copy; Footer ...

      </footer>
    )
  }
}
