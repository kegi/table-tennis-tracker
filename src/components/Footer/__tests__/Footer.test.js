import React from 'react'
import Footer from './../'
import { mount } from 'enzyme'

describe('component:Footer', () => {
  it('render correctly', () => {
    const component = mount(
      <Footer/>
    )
    expect(component).toMatchSnapshot()
  })
})
