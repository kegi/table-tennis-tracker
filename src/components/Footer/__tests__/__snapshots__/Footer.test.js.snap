// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component:Footer render correctly 1`] = `ReactWrapper {}`;
